src/Components/Button.js

```diff
@@ -1,23 +1,34 @@
 import React, { useState } from 'react';
 import * as Icons from '../Components/Icons';
 import classes from '../App.module.css';
+require('typeface-inter');
```

Comment:
Don't mix require and import syntax. They are equivalent, but import syntax is the new es6 standard, so this should be written as `import Inter from "typeface-inter"`

Reply:
However, I think here you should stick to using font faces instead of using a package specifically just to load this font.

You would do that by adding it as a global style with:
```
  @font-face {
    font-family: 'inter-regular';
    src: url('https://slate.textile.io/ipfs/bafkreic3hkcuwvrmf6trweqcjp62valsfjh3zvwacikoreynakgw67wrvy');
  }
```

Fonts can either be loaded from a local file (like the one you have in static) or from a url like the one above. I'd recommend loading from a url like we do in the main app bc then it can load from that url simultaneously while the rest of the app loads from a different server, rather than competing for the same "loading bandwidth" from a single source. This would mean you can delete the Inter-Regular.ttf font from the public folder

Reply:
for security reasons chrome doesn't allow linking to am external url for anything like css, scripts, fonts and all that.